stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -30,21 +30,25 @@ const SecuritiesTable = () => {
 
   return (
     <div className="securities-container">
-      <div className="securities-title">My Securities Table</div>
+      <div className="securities-title-container">
+        <div className="securities-title">My Securities Table</div>
+        <div className="add-securities-button-above-table">
+          {securitiesData.length > 0 && <AddSecurityButton />}
```

Comment:
This doesn't look right. You're using the `AddSecurityButton` as part of a ternary statement where you also compare the length of the `securitiesData`. My guess is that you wanted to check if the number of securities is greater than 0 i.e. there are some securities, and therefore we'll be displaying a table so you'd want a button to be displayed above the table. You probably want something like this:

    {securitiesData.length > 0 ? <AddSecurityButton /> : ''}

Reply:
Following our discussion and talking with Josh, I'm fine for this to stay the way you have it since it seems to be quite common.